### ISSUE TITLE
Verification Information

### ISSUE THREAD

**Issue by Be2021** [sudo-do/discord-selfbot-owo-bot#16]
Cannot recognize the information that prompts for verification

**Comment by sudo-do** [comment]
> Cannot recognize the information that prompts for **verification**

what verification? can you supply a screenshot, please?

**Comment by Be2021** [comment]
Hello, I am glad to hear from you!  There is no prompt for verification in the channel
[Image.jpeg]
获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Nizel ***@***.***>
发送时间: Wednesday, July 28, 2021 11:33:20 PM
收件人: sudo-do/discord-selfbot-owo-bot ***@***.***>
抄送: Be2021 ***@***.***>; Author ***@***.***>
主题: Re: [sudo-do/discord-selfbot-owo-bot] Verification Information (#16)


Cannot recognize the information that prompts for verification

what verification? can you supply a screenshot, please?

―

**Comment by sudo-do** [comment]
attachments on e-mail replies are not visible on github, so, please left a photo URL or reply from github

**Comment by Be2021** [comment]
![124545454545](https://user-images.githubusercontent.com/86850304/127356073-bf202879-7c4a-4a11-bde2-5fc2012de352.jpg)
There is no prompt for verification in the channel

**Comment by sudo-do** [comment]
looks like the security function doesn't work well, idfk why for now, but I'll fix it.

**Comment by sudo-do** [comment]
also related with #13

**Comment by Be2021** [comment]
Yes, as long as there is verification in the channel, a prompt will pop up to remind players

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Nizel ***@***.***>
发送时间: Thursday, July 29, 2021 12:03:45 AM
收件人: sudo-do/discord-selfbot-owo-bot ***@***.***>
抄送: Be2021 ***@***.***>; State change ***@***.***>
主题: Re: [sudo-do/discord-selfbot-owo-bot] Verification Information (#16)


also related with #13<https://github.com/sudo-do/discord-selfbot-owo-bot/issues/13>

―
